WordPress/Classes/Utility/Analytics/WPAnalyticsEvent.swift

```diff
@@ -669,4 +672,25 @@ extension WPAnalytics {
         WPAnalytics.track(stat, withProperties: props)
     }
 
+    /// This will call each registered tracker and fire the given event.
+    /// - Parameters:
+    ///   - eventName: a `String` that represents the event name
```

Comment:
Just a thought. I don't feel strongly.

```suggestion
    ///   - eventName: a `String` that represents the Block Editor event name
```